lxd/cluster/recover.go

```diff
@@ -538,3 +539,87 @@ func unpackTarball(tarballPath string, destRoot string) error {
 
 	return nil
 }
+
+// createTarball creates tarball at tarballPath, rooted at rootDir and including
+// all files in walkDir except those paths found in excludeFiles.
+// walkDir and excludeFiles elements are relative to rootDir.
+func createTarball(tarballPath string, rootDir string, walkDir string, excludeFiles []string) error {
+	tarball, err := os.Create(tarballPath)
```

Comment:
we should add a revert in case the create files part way through to clean up this file